packages/chakra-ui/src/components/ui/radio.tsx

```diff
@@ -0,0 +1,20 @@
+import { RadioGroup as ChakraRadioGroup } from '@chakra-ui/react';
+import * as React from 'react';
```

Comment:
Two things. First, put the react imports up top. Secondly, don't use the `import * as React from 'react';` pattern. Instead import only that which you need. Like:
`import { InputHTMLAttributes, Ref, forwardRef } from 'react'`;